### PR TITLE
fix for #4 and #13 + small things

### PR DESCRIPTION
**Pull request by 1holz**
fix for #4 and #13 + small things like removing some unused imports

### CODE REVIEW

File: src/main/java/io/github/cottonmc/component/data/impl/SimpleUnit.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 public class SimpleUnit implements Unit {
 	public static final NumberFormat FORMAT_STANDARD = NumberFormat.getNumberInstance();
 	static {
-		FORMAT_STANDARD.setMinimumFractionDigits(0);
+		FORMAT_STANDARD.setMinimumFractionDigits(2);
```
Comment (UpcraftLP) [inline src/main/java/io/github/cottonmc/component/data/impl/SimpleUnit.java:10]: I don't think we should *always* force 2 digits.

**Review by UpcraftLP — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/cottonmc/component/data/impl/SimpleUnit.java (hunk 1 above)
Comment (LemmaEOF) [reply src/main/java/io/github/cottonmc/component/data/impl/SimpleUnit.java:10]: This is to fix #4 - check the video they link on the issue. I'm not a *hundred* percent happy with this, to be honest.

**Review by LemmaEOF — APPROVED** [review]

**Review by UpcraftLP — APPROVED** [review]
Aight, guess we can merge this anyway, and if we need a different way to format we can add that later.

**Comment by 1holz** [comment]
so will somebody merge this?

**Comment by UpcraftLP** [comment]
no

**Pull request merged**